client/state/sites/selectors.js

```diff
@@ -56,6 +62,7 @@ export const getSite = createSelector(
 			hasConflict: isSiteConflicting( state, siteId ),
 			slug: getSiteSlug( state, siteId ),
 			domain: getSiteDomain( state, siteId ),
+			is_previewable: isSitePreviewable( state, siteId )
```

Comment:
Not trying to start a style debate, but I've taken to using trailing commas here. IMO it makes diffs easier to grok, and allows painless reordering.